spec.html

```diff
@@ -12066,7 +12079,17 @@ <h1>
       <p>The default implementation of HostCallJobCallback performs the following steps when called:</p>
       <emu-alg>
         1. Assert: IsCallable(_jobCallback_.[[Callback]]) is *true*.
-        1. Return ? Call(_jobCallback_.[[Callback]], _V_, _argumentsList_).
+        1. Let _callerContext_ be the running execution context.
+        1. If _callerContext_ is not already suspended, suspend _callerContext_.
+        1. Let _jobContext_ be a new execution context.
+        1. Let _scriptOrModule_ be _jobCallback_.[[ScriptOrModule]].
+        1. If _scriptOrModule_ is not *null*, then
+          1. Set the Realm of _jobContext_ to _scriptOrModule_.[[Realm]].
+        1. Set the ScriptOrModule of _jobContext_ to _scriptOrModule_.
+        1. Perform any necessary implementation-defined initialization of _jobContext_.
+        1. Let _result_ be Completion(Call(_jobCallback_.[[Callback]], _V_, _argumentsList_)).
+        1. Remove _jobContext_ from the execution context stack and restore _callerContext_ as the running execution context.
```

Comment:
In https://github.com/tc39/ecma262/pull/2962, we change this phrasing. Depending on the order these land, the latter PR will need to account for the former.